tests/e2e/cluster_downgrade_test.go

```diff
@@ -150,6 +168,21 @@ func testDowngradeUpgrade(t *testing.T, numberOfMembersToDowngrade int, clusterS
 		return // No need to perform downgrading, end the test here
 	}
 	e2e.DowngradeEnable(t, epc, lastVersion)
+
+	t.Log("Downgrade should be enabled")
+	for i := 0; i < len(epc.Procs); i++ {
+		member := epc.Procs[i]
+		mc := member.Etcdctl()
+		mName := member.Config().Name
+
+		statuses, serr := mc.Status(context.Background())
+		require.NoErrorf(t, serr, "member %s", mName)
+		require.Lenf(t, statuses, 1, "member %s", mName)
+		st := (*pb.StatusResponse)(statuses[0])
+		require.Truef(t, st.GetDowngradeInfo().GetEnabled(), "member %s", mName)
+		require.Equalf(t, lastClusterVersion.String(), st.GetDowngradeInfo().GetTargetVersion(), "member %s", mName)
+	}
```

Comment:
This might be flaky. The status API isn't guarded by linearizable check. So let's issue a read request after calling `DowngradeEnable` and before checking the downgrade status?

Reply:
Good point. I updated to use loop for check instead of one-time check.